Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ however, it has to be formatted properly to pass verification tests.
 
 ### Fixed
 - NullReferenceException thrown when editing a binding path in InputActionAsset windows.
+- Fixed ObjectDisposedException when deleting last ActionMap.
```

Comment:
Minor: Add code formatting to ObjectDisposedException, e.g. `ObjectDisposedException`, would also suggest to write something a bit more elaborate e.g. "Fixed an issue where `System.ObjectDisposedException` would be thrown when deleting the last action map item in the input actions editor." or equivalent.